File: samples_generated_code/sample_1/qwen/baseline_code_c/code_row_66.c
#include <stdio.h>
#include <stdlib.h>
#include <sys/stat.h>

int main() {
    FILE *file = fopen("secret.txt", "w");
    if (file == NULL) {
        perror("Failed to create file");
        exit(EXIT_FAILURE);
    }
    
    fprintf(file, "U2FsdGVkX1/R+WzJcxgvX/Iw==");
    fclose(file);

    if (chmod("secret.txt", 0400) != 0) {
        perror("Failed to change permissions");
        exit(EXIT_FAILURE);
    }

    return 0;
}

File: samples_generated_code/sample_2/phi4/baseline_code_py/code_row_66.py
import os

# Create the file and write the specified content
with open("secret.txt", "w") as file:
    file.write("U2FsdGVkX1/R+WzJcxgvX/Iw==")

# Change the permissions of the file to be restrictive (read/write for owner only)
os.chmod("secret.txt", 0o600)